pkg/arm/data/azuredeploy.json

```diff
@@ -26,7 +26,93 @@
                 ]
             }
         },
-{{ end }}
+{{- end }}
+{{/* TODO: we may be able to avoid the IP and LB here if we stop using */}}
+{{/* DaemonSets for SDN.  Then Master nodes should come up as Ready */}}
```

Comment:
So how would you deploy SDN in this case?

Reply:
static pods